src/output-json-alert.c

```diff
@@ -390,6 +390,95 @@ static void AlertAddPayload(AlertJsonOutputCtx *json_output_ctx, json_t *js, con
     }
 }
 
+static json_t *JsonBuildFileInfoRecord(const Packet *p, const File *ff)
```

Comment:
Would like to avoid this duplication with `JsonBuildFileInfoRecord` from output-json-file.c

Reply:
I plan to refactor as much as possible. This was just a POC implementation.